contrib/containers/ci/Dockerfile

```diff
@@ -42,12 +40,37 @@ RUN apt-get update && apt-get install $APT_ARGS \
     libxcb-xinerama0 \
     libxcb-xkb1 \
     libxkbcommon-x11-0 \
-    wget \
+    lsb-release \
+    software-properties-common \
     unzip \
+    wget \
     m4 \
     pkg-config \
     zlib1g-dev
 
+# Install Clang+LLVM and set it as default
+# We don't need all packages but the default set doesn't include some
+# packages we want so we will need to install some of them manually.
+ARG LLVM_VERSION=16
+RUN set -ex; \
```

Comment:
looks a bit too complex, maybe better to make sh script for it instead line for better readability?

Reply:
Making a script that won't be reused/user-triggerable doesn't seem necessary, but I can add `echo` comments to serve as documentation for steps. Would that be alright?